tests/unit/models/test_ledger_object.py

```diff
@@ -378,6 +483,7 @@ def test_account_root(self):
         expected = AccountRoot(
             index="13F1A95D7AAB7108D5CE7EEAF504B2894B8C674E6D68499076441C4837282BF8",
             account="rf1BiGeXwwQoi8Z2ueFYTEXSwuJYfV2Jpn",
+            amm_id=None,
```

Comment:
Clarification: Is it necessary for an `AccountRoot` json to contain the `amm_id` and `first_nftoken_sequence` field? These are specified as `Optional` in the class definition.

The unit and integration tests work fine without these fields, I just want confirmation.

Reply:
Not necessary, I can remove them.